### DISCUSSION TITLE
ipv6 hosts in proxy

### DISCUSSION THREAD

**Discussion by hellgod77** [ZoeyVid/NPMplus#752] [Q&A] [answered]
![grr2](https://github.com/ZoeyVid/NPMplus/assets/76710563/94db5303-cc54-436a-8358-987cca0b5201)
if you put a ip6 addy here it will just bring a error 500 page 
![temp](https://github.com/ZoeyVid/NPMplus/assets/76710563/8cc709ca-e4c9-44e1-8dde-b0b983001e02)
but if i make aaaa record to it and point to the domain it works.. this is time consuming is there any other way? the original npm did it also

**Comment by Zoey2936** [comment, marked as answer]
your ipv6 adress needs to be sorrunded by []

**Comment by hellgod77** [comment]
ya, again with face palm and feeling like a idiot, thank you
